src/sqliteInt.h

```diff
@@ -680,7 +680,11 @@
 # ifndef SQLITE_BIG_DBL
 #   define SQLITE_BIG_DBL (((sqlite3_int64)1)<<50)
 # endif
+#ifndef FREEBSD_KERNEL
 # define SQLITE_OMIT_DATETIME_FUNCS 1
+#else
+ #define SQLITE_OMIT_LOCALTIME
```

Comment:
isn't this already defined in line 352 of the configure.ac?
Maybe we can just remove this whole if and add at line 674 of this file
```
# define SQLITE_OMIT_DATETIME_FUNCS
```
what do you think?